source/blender/io/ply/exporter/ply_export_header.cc

```diff
@@ -18,36 +18,44 @@ void write_header(std::unique_ptr<FileBuffer> &buffer,
                   const PLYExportParams &export_params)
 {
   buffer->write_string("ply");
+
   StringRef format = export_params.ascii_format ? "ascii" : "binary_little_endian";
   buffer->write_string("format " + format + " 1.0");
+
   StringRef version = BKE_blender_version_string();
   buffer->write_string("comment Created in Blender version " + version);
+
   buffer->write_header_element("vertex", int32_t(plyData->vertices.size()));
   buffer->write_header_scalar_property("float", "x");
   buffer->write_header_scalar_property("float", "y");
   buffer->write_header_scalar_property("float", "z");
+
   if (!plyData->vertex_normals.is_empty()) {
     buffer->write_header_scalar_property("float", "nx");
     buffer->write_header_scalar_property("float", "ny");
     buffer->write_header_scalar_property("float", "nz");
   }
+
   if (!plyData->vertex_colors.is_empty()) {
     buffer->write_header_scalar_property("uchar", "red");
     buffer->write_header_scalar_property("uchar", "green");
     buffer->write_header_scalar_property("uchar", "blue");
     buffer->write_header_scalar_property("uchar", "alpha");
   }
+
   if (!plyData->UV_coordinates.is_empty()) {
     buffer->write_header_scalar_property("float", "s");
     buffer->write_header_scalar_property("float", "t");
   }
+
   buffer->write_header_element("face", int32_t(plyData->faces.size()));
   buffer->write_header_list_property("uchar", "uint", "vertex_indices");
```

Comment:
Maybe also surround this with an if-statement, in case of no faces at all.
```suggestion
if (!plyData->faces.is_empty()) {
  buffer->write_header_element("face", int32_t(plyData->faces.size()));
  buffer->write_header_list_property("uchar", "uint", "vertex_indices");
}
```